lib/mapping/CCampaignHandler.cpp

```diff
@@ -98,27 +80,29 @@ std::unique_ptr<CCampaign> CCampaignHandler::getCampaign( const std::string & na
 
 		std::string scenarioName = name.substr(0, name.find('.'));
 		boost::to_lower(scenarioName);
-		scenarioName += ':' + boost::lexical_cast<std::string>(g-1);
+		scenarioName += ':' + std::to_string(g - 1);
 
 		//set map piece appropriately, convert vector to string
 		ret->mapPieces[scenarioID].assign(reinterpret_cast< const char* >(file[g].data()), file[g].size());
 		CMapService mapService;
-		ret->scenarios[scenarioID].scenarioName = mapService.loadMapHeader((const ui8*)ret->mapPieces[scenarioID].c_str(), (int)ret->mapPieces[scenarioID].size(), scenarioName)->name;
+		ret->scenarios[scenarioID].scenarioName = mapService.loadMapHeader(reinterpret_cast<const ui8 *>(ret->mapPieces[scenarioID].c_str()),

```

Comment:
Something went wrong with formatting here. 
For long methods I think we should use
```
someVeryLongMethodName(
<+1 tab> parameter1,
<+1 tab> parameter2 );
```

Reply:
Done with help of helper variable and your suggestion.